browser-extension/mv2/src/background/background.js

```diff
@@ -560,9 +560,53 @@ BG.Methods.onBeforeRequest = (details) => {
   return BG.Methods.modifyUrl(details);
 };
 
+BG.Methods.modifyHeadersForSessionReplayPlayer = ({ updatedHeaders, modifiedHeaders, details, originalHeaders }) => {
```

Comment:
```suggestion
BG.Methods.modifyHeadersForSessionReplayPlayer = ({ headersToUpdate, ruleModifiedHeaders, requestDetails, originalHeaders }) => {
```

Or something more relevant. `updatedHeaders` is confusing. `modifiedHeaders` is difficult to understand if you don't have the context